x-pack/plugins/security_solution/server/usage/endpoints/fleet_saved_objects.ts

```diff
@@ -29,9 +37,11 @@ export const getLatestFleetEndpointEvent = async (
   agentId: string
 ) =>
   savedObjectsClient.find<AgentEventSOAttributes>({
+    // Get the most recent endpoint event.
```

Comment:
@oatkiller, after speaking with @rudolf yesterday, there doesn't seem to be a way we can really batch this. I.e. we cannot say "given a list of agent id's, give me the most recent event for each id". 

Thinking about it more, we may be able to say, "give me every event for each of these agent id's using an `or` in the filter, but that would be a massive response to then filter". This will definitely need perf testing